tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -79,6 +79,20 @@ def run(self):
                     {"message": "Failed to sync cluster network details"}
                 )
 
+        if NS.tendrl_context.integration_id:
+            # Initialize alert node alert count
+            try:
+                key = 'clusters/%s/nodes/%s/alert_counters' % (
```

Comment:
use 
if NS.tendrl.objects.ClusterNodeAlertCounters(
                        node_id=NS.node_context.node_id,
                        integration_id=NS.tendrl_context.integration_id
                   ).exists()

Reply:
i will change this